packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -10,27 +27,30 @@
     cursor-pointer
     transition-default
     focus-base
-    no-underline
-    text-0h;
-  color: inherit;
-  border-block-end: 2px solid transparent;
+    no-underline;
+  color: var(--calcite-navigation-logo-text-color, inherit);
+  font-size: var(--calcite-font-size-md);
+  line-height: var(--calcite-font-line-height-fixed-lg);
+  border-block-end: var(--calcite-spacing-base) solid
+    var(--calcite-navigation-logo-background-color, var(--calcite-color-foreground-1));
```

Comment:
should this be `--calcite-navigation-logo-border-color`?

Reply:
Yup , good catch.